src/SUMMARY.md

```diff
@@ -22,6 +22,9 @@
 * [Resolving domains in a browser](browser-resolution/resolving-domains-in-a-browser.md)
 * [Browser resolution algorithm](browser-resolution/browser-resolution-algorithm.md)
 
+## Integrations
+* [Getting Started](integrations/getting-started.md)
```

Comment:
```suggestion
* [Getting started](integrations/getting-started.md)
```

We follow these capitalization rules: https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings